### ISSUE TITLE
Don't label PRs with moved expressions with `has: package(new)`

### ISSUE THREAD

**Issue by Ma27** [NixOS/ofborg#116]
Have a look at the following PR for instance: https://github.com/NixOS/nixpkgs/pull/36934

This PR did a minor rename change, however the PR was labeled to contain a new package which might be a bit misleading especially when the PR aims to fix a package that might be relevant for the upcoming release.

**Comment by grahamc** [comment]
Hmm so the root of the issue is: https://github.com/NixOS/nixpkgs/pull/36934/files#diff-2afbba83e1b14f9e0c304c769831a0e2R232 the code sees that a new attribute has been created and thinks it is a new package.

Perhaps what could be done is determining if the outpath of the new package is known by other attributes as well...

**Comment by Ma27** [comment]
Makes sense. Please keep in mind that I might lack time to find a suitable solution ATM (busy with exams actually :/), but I'll poke you if I find some to start work to avoid duplicated efforts :)
